src/Microsoft.VisualStudio.Services.Agent/HostContext.cs

```diff
@@ -244,6 +244,7 @@ public Tracing GetTrace(string name)
 
             // Create a new instance.
             T svc = Activator.CreateInstance(target) as T;
+
```

Comment:
not sure whether we need the extra line. :)

Reply:
My code won't work without it! :D